.github/workflows/release.yml

```diff
@@ -4,13 +4,16 @@ on:
   release:
     types: [published]
 
+permissions:
+  contents: read
```

Comment:
Remember we had this problem with schemas repo where we needed write access to create a zip file to download, don't we need that kind of access here as well?

Reply:
Updated to have write permissions too 👍